spaceship/lib/spaceship/test_flight/build.rb

```diff
@@ -71,12 +71,12 @@ class Build < Base
       export_compliance_missing: 'testflight.build.state.export.compliance.missing'
     }
 
-    # Find a Build by `build_id`. Returns `nil` if can't find it.
+    # Find a Build by `build_id`.
     #
     # @return (Spaceship::TestFlight::Build)
     def self.find(app_id: nil, build_id: nil)
       attrs = client.get_build(app_id: app_id, build_id: build_id)
-      self.new(attrs) if attrs
+      self.new(attrs)
```

Comment:
`initialize` always expects a hash. If `nil` is passed in, it throws an exception. So far the pattern has been that if an error response is received, an `UnexpectedResponse` exception is thrown. I think throwing exceptions is consistent in this case.